### PR TITLE
Listings with rows count

### PR DESCRIPTION
**Pull request by user-yormen**
#1634

### CODE REVIEW

File: sormas-api/src/main/resources/strings.properties [hunk 1]
```diff
@@ -665,6 +665,15 @@ notificationTaskStartSubject = Task to be started
 notificationVisitCompleted = A follow-up visit for contact %s assigned to user %s has been completed.
 notificationVisitCompletedSubject = Follow-up visit completed
 
+#Labels
+labelNumberOfUsers = No. of users
+labelNumberOfRegions = No. of regions
+labelNumberOfDistricts = No. of districts
+labelNumberOfCommunities = No. of communities
+labelNumberOfPointofEntry = Points of entry
```
Comment (MateStrysewskeSym) [inline sormas-api/src/main/resources/strings.properties:673]: Should be "No. of points of entry" for consistency

File: sormas-ui/src/main/java/de/symeda/sormas/ui/configuration/infrastructure/AbstractFacilitiesView.java [hunk 2]
```diff
@@ -314,6 +319,29 @@ public void run() {
...
+	public HorizontalLayout createRowCountLayout() {
+		HorizontalLayout totalLayout = new HorizontalLayout();
+		totalLayout.setMargin(false);
+		totalLayout.addStyleName(CssStyles.VSPACE_4);
+		totalLayout.setSpacing(true);
+		totalLayout.setWidth(100, Unit.PERCENTAGE);
+
+		String labelText = FacilityType.LABORATORY.equals(criteria.getType()) ? Strings.labelNumberOfLaboratories : Strings.labelNumberOfFacilities;
+
+		Label labelTotal = new Label(I18nProperties.getString(labelText) + ":");
```
Comment (MateStrysewskeSym) [inline sormas-ui/src/main/java/de/symeda/sormas/ui/configuration/infrastructure/AbstractFacilitiesView.java:331]: Naming is a bit unclear, rename this to something like "labelCaption"

File: sormas-ui/src/main/java/de/symeda/sormas/ui/configuration/infrastructure/AbstractFacilitiesView.java [hunk 3]
```diff
@@ -314,6 +319,29 @@ public void run() {
...
+		labelTotal.addStyleNames(CssStyles.LABEL_BOLD, CssStyles.VSPACE_TOP_NONE);
+		totalLayout.addComponent(labelTotal);
+		totalLayout.setExpandRatio(labelTotal, 1);
+		totalLayout.setComponentAlignment(labelTotal, Alignment.MIDDLE_RIGHT);
+
+		Label totalLabelValue = new Label(String.valueOf(grid.getItemCount()));
```
Comment (MateStrysewskeSym) [inline sormas-ui/src/main/java/de/symeda/sormas/ui/configuration/infrastructure/AbstractFacilitiesView.java:337]: See above, I'd simply suggest "labelValue"

**Comment by MateStrysewskeSym** [comment]
Thank you!

**Comment by user-yormen** [comment]
Thank you

**Pull request merged**